centreon/www/front_src/src/Header/Poller/usePollerDatas.ts

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, useMemo } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+import { useAtomValue } from 'jotai/utils';
+import { useTranslation } from 'react-i18next';
+import { equals, isNil } from 'ramda';
+
+import { useFetchQuery } from '@centreon/ui';
+import { refreshIntervalAtom, userAtom } from '@centreon/ui-context';
+
+import useNavigation from '../../Navigation/useNavigation';
+import { pollerListIssuesEndPoint } from '../api/endpoints';
+import { pollerIssuesDecoder } from '../api/decoders';
+
+import { getPollerPropsAdapter } from './getPollerPropsAdapter';
+import type { GetPollerPropsAdapterResult } from './getPollerPropsAdapter';
+
+interface UsePollerDatasResust {
+  data: GetPollerPropsAdapterResult;
+  error: unknown;
+  isAllowed: boolean;
+  isLoading: boolean;
+}
+
+export const usePollerDatas = (): UsePollerDatasResust => {
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { allowedPages } = useNavigation();
+  const { isExportButtonEnabled } = useAtomValue(userAtom);
+  const refetchInterval = useAtomValue(refreshIntervalAtom);
+  const [datas, setDatas] = useState(null);
+  const [isAllowed, setIsAllowed] = useState<boolean>(true);
```

Comment:
It's weird. The hooks sort plugin must be applied as an error here. Because useState must come before useAtomValue

Reply:
they both conflict themself, when I change the order, it ask me to change it back and so forth

Reply:
What are the impacted hooks?